format_spec/history.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Format version history
+---
+
+# Format Version History
+
+## Version 22
+
+Introduced in TileDB 2.25
+
+* The _Current domain_ field was added to [array schemas](./array_schema.md).
+
+## Version 21
+
+Introduced in TileDB 2.19
+
+* The TileDB implementation has been updated to fix computing [tile metadata](./fragment.md#tile-mins-maxes) for nullable fixed-size strings on dense arrays.
+
+> [!NOTE]
+> This version does not contain any changes to the storage format, but was introduced as an indicator for implementations to not rely on tile metadata under certain circumstances on previous versions.
+
+## Version 20
+
+Introduced in TileDB 2.17
+
+* Arrays can have [enumerations](./enumeration.md).
+* The bit-width reduction and positive delta filters are supported on data of date or time types.
+* The [filter pipeline options](./filter_pipeline.md#filter-options) for the double-delta filter contain the _Reinterpret datatype_ field.
+
+## Version 19
+
+Introduced in TileDB 2.16
+
+* [Vacuum files](./vacuum_file.md) contain relative paths to the location of the array.
+* The [filter pipeline options](./filter_pipeline.md#filter-options) for the delta filter contain the _Reinterpret datatype_ field.
+
+## Version 18
+
+Introduced in TileDB 2.15
+
+* Arrays can have [dimension labels](./array_schema.md#dimension-label).
+
+## Version 17
+
+Introduced in TileDB 2.14
+
+* The _Order_ field was added to [attributes](./array_schema.md#attribute).
+* Cell offsets in dimensions or attributes of UTF-8 string type are not written in the offset tiles, if the RLE or dictionary filter exists in the filter pipeline. They are instead encoded as part of the data tile.
```

Comment:
#3868

---

I am not entirely sure, from the storage format perspective does this change only mean that the offsets file will be empty?

Reply:
I think so